src/renderers/dom/client/eventPlugins/SimpleEventPlugin.js

```diff
@@ -156,7 +156,7 @@ function isInteractive(tag) {
   );
 }
 
-function shouldPreventMouseEvent(inst) {
+function shouldPreventMouseEvent(inst?: SyntheticEvent) {
```

Comment:
This isn't a `SyntheticEvent`. The type we have flowing from where this is called in `extractEvents` says this should be `ReactInstance`